drivers/scheduler/k8s/k8s.go

```diff
@@ -616,6 +616,72 @@ func (k *k8s) GetNodesForApp(ctx *scheduler.Context) ([]node.Node, error) {
 	return result, nil
 }
 
+func (k *k8s) GenerateSupportBundle(ctx *scheduler.Context) (string, error) {
+	var output string
```

Comment:
Strings in golang are immutable. So this routine is going to allocate a new string each time you have `output +=`.

Use a byte array. Reference: https://stackoverflow.com/questions/1760757/how-to-efficiently-concatenate-strings-in-go

Reply:
Done